TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java

```diff
@@ -0,0 +1,57 @@
+package com.ybsystem.tweetmate.fragments.dialog;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.view.WindowManager;
+
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.fragments.timeline.ProfileTimeline;
+import com.ybsystem.tweetmate.models.entities.Column;
+
+import static com.ybsystem.tweetmate.models.enums.ColumnType.USER_TWEET;
+
+public class PreviewAppearanceDialog extends DialogFragment {
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        // Show full screen
+        Window window = getDialog().getWindow();
+        WindowManager.LayoutParams params = window.getAttributes();
+        params.width = ViewGroup.LayoutParams.MATCH_PARENT;
+        params.height = ViewGroup.LayoutParams.MATCH_PARENT;
+        window.setAttributes(params);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate
+        View view = inflater.inflate(R.layout.dialog_preview_appearance, null);
+
+        // Declare
+        long TwitterJpUserId = 7080152L;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS00117)